src/routes/apiDirectory.js

```diff
@@ -20,9 +22,40 @@ authMiddleware.unless = unless;
 
 const router = express.Router();
 
+router.use(httpContext.middleware);
 router.use(cookieSession);
 router.use(authMiddleware.unless({ path: [join('/api', loginPath)] }));
 
+router.use((req, res, next) => {
+  const getSessionSig = (cookie) => {
+    try {
+      if (cookie !== undefined && (typeof cookie === 'string' || cookie instanceof String)) {
+        const sessionSigs = cookie.split(' ').filter((s) => s.includes('session.sig='));
+        return sessionSigs.length > 0
+          ? sessionSigs[0].replace('session.sig=', '')
+          : '';
```

Comment:
Not sure if it'll mess things up but if I have more than one cookie from localhost:3000, the database entry for session.sig has a semicolon

Reply:
hmmm, can you post your value as a comment here?